masterlist.yaml

```diff
@@ -5937,12 +5938,6 @@ plugins:
       - 'Thunderchild - Epic Shout Package.esp'
       - 'TKAA.esp'
     msg:
-      - type: warn
-        condition: 'active("sm_jobasha_bookshop.esp")'
-        content: 'Jobasha''s Books SE is only compatible with its Winterhold Restored version.'
-        subs:
-          - 'Jobasha''s Books SE'
-          - '[Jobasha''s Books and Winterhold Restored Patch](https://www.nexusmods.com/skyrimspecialedition/mods/13285/)'
```

Comment:
This should work, if you're using `subs` the there need to be a ref `%1%' to consume it.
```yaml
    msg:
      - type: warn
        content: 'Use the Winterhold Restored version of Jobasha''s Books SE instead: %1%'
        subs:
          - '[Jobasha''s Books and WinterHold Restored Patch] (https://www.nexusmods.com/skyrimspecialedition/mods/13285/)'
        condition: 'active("sm_jobasha_bookshop.esp")'
```